tests/sap/aibus/dar/client/test_data_manager_client.py

```diff
@@ -679,7 +679,7 @@ def prepare_client(dar_url: str, clazz):
     mock_session = create_autospec(DARSession, instance=True)
     mock_session.get_from_endpoint.return_value = mock_response
     mock_session.post_to_endpoint.return_value = mock_response
-
+    mock_session.post_to_url.return_value = mock_response
```

Comment:
Why do we need this mocK?

Reply:
`create_inference_request_with_url` uses this method underneath. So this mock was added

Reply:
It also took me a while to figure it out, but I got it now. `prepare_client` is used in `test_inference_client.py` to create an inference client with a mocked session (see [here](https://github.com/SAP/data-attribute-recommendation-python-sdk/blob/c1a072a572c0d3fa3e66fe577d4ba80243ebc457/tests/sap/aibus/dar/client/test_inference_client.py#L27)). Therefore, we need this change.

Reply:
Ok, thank you for the explanation!